samples_project/Assets/SampleViewer/Samples/Measure/Measure.unity

```diff
@@ -2192,9 +2192,9 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   Line: {fileID: 820472294}
-  GeodeticDistanceText: {fileID: 134436593}
-  LineMarker: {fileID: 5022259574890728151, guid: 4a39d5da76a48014d93cc6059717dc44, type: 3}
-  InterpolationMarker: {fileID: 8846720079027601179, guid: dcfd660509e86354896380205354f1e6, type: 3}
+  GeodeticDistanceText: {fileID: 0}
+  LineMarker: {fileID: 5022259574890728151, guid: c71f2b8d1ec452a4390a12246e65c700, type: 3}
+  InterpolationMarker: {fileID: 8846720079027601179, guid: e62483263eba74b4b8b1786109d7105c, type: 3}
```

Comment:
What changes were saved here? Do they need to be updated?

Reply:
This change signifies the alteration of the prefabs that are spawned from the measure script. Before the prefabs were shared between the measure and routing samples, I have altered them to utilize prefabs that are specific to the measure sample!